File: app/src/main/java/com/example/client/screens/wallet/fragment/IWalletView.java
package com.example.client.screens.wallet.fragment;

import com.example.client.models.message.MessageModel;
import com.example.client.models.profile.ProfileModel;
import com.example.client.models.transaction.TransactionModel;

import java.util.List;

public interface IWalletView {
    void showInfoUser(ProfileModel user);
    void showTransaction(List<TransactionModel> items,String method);
    void showTransactionByOrderCode(TransactionModel item);
    void deleteRecharge(MessageModel message);
    void requestRecharge(MessageModel message);
    void refreshUserActive(MessageModel message);
    void showLoading();
    void hideLoading();
}

File: app/src/main/java/com/example/client/screens/wallet/fragment/IWalletView.kt
package com.example.client.screens.wallet.fragment

import com.example.client.base.IBaseCollectionView
import com.example.client.models.order.OrderModel
import com.example.client.models.profile.ProfileModel
import com.example.client.models.transaction.TransactionModel

interface IWalletView: IBaseCollectionView {
    fun showWallet(profile: ProfileModel)
    fun showData(items: List<TransactionModel>)
    fun showMoreData(items: List<TransactionModel>)
    fun showEmptyData()
    fun refreshOrderInfo()
    fun showErrorMessage(errorMessage: Int)
    fun updateData(orderModel: OrderModel)
}